heliopy/data/util.py

```diff
@@ -1165,3 +1257,358 @@ def dtime2doy(dt):
         Day of year
     """
     return int(dt.strftime('%j'))
+
+
+def cdf2xr(cdf, index_key, starttime=None, endtime=None, list_keys=None,
+           dtimeindex=True, badvalues=None, ignore=None):
+    """
+    Converts cdf file of spacecraft timeseries data to an xarray (DataArray or
+    Dataset) object. xarray package is used as particle distribution functions
+    are usually multidimensional (3/4D) datasets *f(time, energy, theta, phi).
```

Comment:
```suggestion
    are usually multidimensional (3/4D) datasets *f(time, energy, theta, phi)*.
```
I think this missing asterisk is the only thing breaking the docs build

Reply:
Thanks for the help. I will check it.